### ISSUE TITLE
Getting error in building webpack

### ISSUE THREAD

**Issue by prashantism2209** [bertho-zero/react-redux-universal-hot-example#408]
I am getting below error while building webpack.

ERROR in ./node_modules/bootstrap-loader/lib/bootstrap.loader.js!./node_modules/bootstrap-loader/no-op.js
Module build failed: Error: Cannot find module '/Users/prashantkumar/Desktop/other/webapp-react/node_modules/bootstrap-sass/package.json/package.json'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function._module2.default._resolveFilename (/Users/prashantkumar/Desktop/other/webapp-react/node_modules/require-hacker/babel-transpiled-modules/require hacker.js:442:34)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at exports.default (/Users/prashantkumar/Desktop/other/webapp-react/node_modules/bootstrap-loader/lib/utils/checkBootstrapVersion.js:8:20)
    at Object.module.exports.pitch (/Users/prashantkumar/Desktop/other/webapp-react/node_modules/bootstrap-loader/lib/bootstrap.loader.js:160:65)
 @ ./node_modules/bootstrap-loader/loader.js 2:17-61
 @ multi bootstrap-loader ./src/client.js

**Comment by iKonrad** [comment]
I'm having similar problem when running `npm install`:

```
ERROR in ./node_modules/bootstrap-loader/lib/bootstrap.loader.js!./node_modules/bootstrap-loader/no-op.js
[0] Module build failed: Error: Cannot find module '/Users/konrad/Projects/ElementX/eurosender-frontend/node_modules/bootstrap-sass/package.json/package.json'
[0]     at Function.Module._resolveFilename (module.js:469:15)
[0]     at Function._module2.default._resolveFilename (/Users/konrad/Projects/ElementX/eurosender-frontend/node_modules/require-hacker/babel-transpiled-modules/require hacker.js:442:34)
```

**Comment by lemonCMS** [comment]
Just to help out, This has  todo with recently released version of resolve 1.7.0 To fix this, add the package resolve version 1.6.0 in package.json. Make sure it is pinned on that version.
More info over at #[bootstrap-loader](https://github.com/shakacode/bootstrap-loader/issues/341)
````json
"resolve": "1.6.0"
````

**Comment by prashantism2209** [comment]
@lemonCMS It solved the issue. Thanks
